src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt

```diff
@@ -9,6 +9,7 @@ import com.terraformation.backend.importer.processCsvFile
 import jakarta.inject.Named
 import java.io.InputStream
 import java.time.InstantSource
+import org.hsqldb.Tokens.POSITION
```

Comment:
I think this import should be removed, perhaps it was imported before jOOQ updated the `MODULES` table generated class?